docs/redirects/jdbc.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Why are these redirects added?

Reply:
Just for uniformity with other pages.

Reply:
The redirects are only there because previously the documentation structure was different. For new pages we don't need redirects.